From e3ae56cf0bb4063c31295f45d04e3c504f4b6cc7 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 24 Apr 2017 20:41:25 -0700
Subject: [PATCH] autofs: Upgrade to 5.1.2 release

Use pkg-config first to look for external libnsl which is now
split out from glibc, if it does not exist then see if its provided
by glibc itself.

-Khem

---
 configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index 9cf73b8..1016c71 100644
--- a/configure.in
+++ b/configure.in
@@ -186,7 +186,7 @@ fi
 #
 # glibc/libc 6 new libraries
 #
-AC_CHECK_LIB(nsl, yp_match, LIBNSL="-lnsl")
+PKG_CHECK_MODULES([NSL], [libnsl], [], [AC_CHECK_LIB([nsl],[yp_match],[LIBNSL="-lnsl"],[LIBNSL=""])])
 AC_SUBST(LIBNSL)
 
 AC_CHECK_LIB(resolv, res_query, LIBRESOLV="-lresolv")
